README.md

```diff
@@ -1,226 +1,188 @@
-tx_hash = sdk.create_account('address')
+from kin import AccountStatuses
+
+status = client.get_account_status('address')
 
-# create a new account prefunded with a specified amount of native currency (lumens).
-tx_hash = sdk.create_account('address', starting_balance=1000)
+# status can be one of:
+AccountStatuses.NOT_CREATED
+AccountStatuses.NOT_ACTIVATED
+AccountStatuses.ACTIVATED
 ```
 
-### Checking if Account is Activated (Trustline established)
+
+### Getting Transaction Data
 ```python
-# check if KIN is trusted by some account
-kin_trusted = sdk.check_account_activated('address')
+# Get information about a specific transaction
+# The 'simple' flag is enabled by defualt, and dectates what object should be returned
+# For simple=False: A 'kin.TransactionData' object will return,
+# containig many fields that may be confusing and of no use to the user.
+
+# For simple=True: A 'kin.SimpleTransaction' object will return,
+# containing only the data that the user will need.
+# However, if the transaction if too complex to be simplified, a 'CantSimplifyError' will be raised
+tx_data = sdk.get_transaction_data(tx_hash, simple=True/False)
+
+# A transaction will not be simplifed if:
+# 1. It contains a memo that is not a text memo
+# 2. It contains multiple operations
+# 3. It contains a payment that is not of KIN/XLM
+# 4. It contains activation to anything other than KIN
+# 5. Its operation type is not one of 'Payment'/'Activation'/'Create account'.
+
+# Given the use case of our blockchain, and the tools that we currently provied to interact with it, these conditions should not occur.
 ```
 
-### Sending Currency
+### Verify Kin Payment
+This method provides an easy way to verify that a transaction is what you expect it to be  
 ```python
-# send native currency (lumens) to some address
-tx_hash = sdk.send_native('address', 100, memo_text='order123')
+client.verify_kin_payment('tx_hash','sender','destination',amount,memo(optional),check_memo=True/False)
 
-# send KIN to some address
-tx_hash = sdk.send_kin('address', 1000, memo_text='order123')
-```
+#Lets say that addr1 payed 15 KIN to add2, with the memo 'Enjoy!'
 
-### Getting Transaction Data
-```python
-# create a transaction, for example a new account
-tx_hash = sdk.create_account('address')
-# get transaction data, returns kin.TransactionData
-tx_data = sdk.get_transaction_data(tx_hash)
+client.verify_kin_payment('tx_hash','addr1','addr2',15,'Enjoy!',True) >> True
+client.verify_kin_payment('tx_hash','addr1','addr2',15,'Hello',True) >> False
+client.verify_kin_payment('tx_hash','addr1','addr2',15) >> True
+client.verify_kin_payment('tx_hash','addr1','addr2',10) >> False
+client.verify_kin_payment('tx_hash','addr1','addr3',10) >> False
 ```
 
-### Transaction Monitoring
+### Kin Payment Monitoring
 ```python
-# define a callback function that receives an address and a kin.TransactionData object
+# This method allows you to monitor the kin payments that an account is reciving/sending
```

Comment:
what method? maybe introduce it, also worth also moving to markdown instead of comment